ext/gmp/gmp.c

```diff
@@ -339,7 +339,24 @@ static zend_object *gmp_clone_obj(zend_object *obj) /* {{{ */
 /* }}} */
 
 static void shift_operator_helper(gmp_binary_ui_op_t op, zval *return_value, zval *op1, zval *op2, uint8_t opcode) {
-	zend_long shift = zval_get_long(op2);
+	bool failed = true;
+	zend_long shift = zval_try_get_long(op2, &failed);
+	if (failed) {
+		//char operator_sigil[2];
+		char *operator_sigil;
```

Comment:
Should be const

Reply:
Right, I initially tried to do some stuff with the fixed width char array to be const but my IDE was complaining.